pkgs/applications/misc/blender/default.nix

```diff
@@ -14,10 +14,11 @@ with lib;
 let python = pythonPackages.python; in
 
 stdenv.mkDerivation rec {
-  name = "blender-2.79b";
+  pname = "blender";
+  version = "2.80";
 
   src = fetchurl {
-    url = "https://download.blender.org/source/${name}.tar.gz";
+    url = "https://download.blender.org/source/${pname}-${version}.tar.gz";
     sha256 = "1g4kcdqmf67srzhi3hkdnr4z1ph4h9sza1pahz38mrj998q4r52c";
```

Comment:
Don't seem to be able to download the file even though it exists?

Reply:
Looks okay here, probably just blender.org being hammered due to the release announcement. What with them being at SIGGRAPH to show it off and all ^.^